app/javascript/packages/verify-flow/steps/personal-key-confirm/personal-key-input.tsx

```diff
@@ -53,6 +59,7 @@ function PersonalKeyInput(
         onInit={(owner) => {
           (owner as ReactInstanceWithCleave & CleaveInstanceInternalAPI).updateValueState();
         }}
+        value={initialValue}
```

Comment:
Looking at how this is assigned and [how Cleave updates in response to a changing value](https://github.com/nosir/cleave.js/blob/e3fa6f3637fa6d8ea97ef66ac1957950632b72ab/src/Cleave.react.js#L26), this looks like a [controlled input](https://reactjs.org/docs/forms.html#controlled-components), which should be fine as-is, but I'd expect the prop to be named `value`, not `initialValue`.

Reply:
Ah, I see. willfix